### ISSUE TITLE
The emails stream provides all historical data on each extraction

### ISSUE THREAD

**Issue by clorichel** [singer-io/tap-yotpo#3]
Stitch users here. Might have missed something, but it seems like the `emails` stream does not limit itself to the bookmark date, which leads to an increasing number of lines extracted and loaded, instead of just the one that changed.

An email "event" with no change at all (no more clicks, no more open, nothing different than last time seen) is consistently added as a new line, when we expect a new line only when there is an update for this email.

Obviously, we built BigQuery views to account for the append-only strategy and that doesn't interfere with our final filtered data. But the amount of lines added for no reason at all is so huge that we'd love to get it down 🤓

**Comment by tbosilevac** [comment]
We have expierenced the same thing.  It in fact affected our licensing for a bit.  I have reduced our frequency to 24 hours and that has helped significantly.   

While this was for Klaviyo, I think the issue to be the same, this is from Stitch support:  
That being said, most tables in this integration use Key Based Incremental replication where we hold a max value bookmark for the tables replicated, and replicate the equal to or greater than value on the next extraction. However, the Lists table is replicated in full each time.
